src/lib/POWERMGNT/POWERMGNT.cpp

```diff
@@ -129,6 +129,12 @@ PowerLevels_e POWERMGNT::setPower(PowerLevels_e Power)
         Power = PWR_100mW;
         break;
     }
+
+#elif defined(TARGET_RX_MATEK_2400)
```

Comment:
No need to define target flag here.
And move this block to first. This way same procedure can be used for any RX with custom default power.